docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -5,9 +5,11 @@
 Adding modules and plugins locally
 **********************************
 
-The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by including it in a collection or embedding it in a role, then publishing the collection or role on Ansible Galaxy. If you are using roles on Ansible Galaxy, then you are already using local modules and plugins without realizing it.
+The easiest, quickest, and the most popular way to extend Ansible is to use a local module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by including it in a collection, then publishing the collection on Ansible Galaxy. If you are using collections from Ansible Galaxy, then you are already using local modules and plugins without realizing it.
```

Comment:
> If you are using collections from Ansible Galaxy, then you are already using local modules and plugins without realizing it.

I don't think this is correct; I think "local" refers to modules/plugins outside ansible-core and outside collections. (The previous version wasn't much better though, since roles from galaxy do not necessarily include modules/plugins.) (And I have no idea whether modules/plugins in local collections count as local modules/plugins...)

Reply:
addressed in c3790e3887d1eb3d7b3437f5ec4794233f1ff03d